test/performance-tests/run-performance-test.py

```diff
@@ -226,72 +241,127 @@ def makePerformancePlot(title, listOfMethods):
 
 # Create the parameter file.
 parameterFileName = "crux.param"
-createParameterFile(parameterFileName)
+# createParameterFile(parameterFileName)
+
+# # Create the index for diameter.
```

Comment:
Are these commented lines necessary to keep?

Reply:
I still want to update the performance tests. But I think, I'll do this later in the next PR.  This is just a performance test. It works fine for now.